app/components/Nav/Main/MainNavigator.js

```diff
@@ -441,6 +441,11 @@ const FiatOnRampAggregator = () => (
         component={Region}
         options={{ animationEnabled: false }}
       />
+      <Stack.Screen
```

Comment:
I do understand we need some routes here with disabled animations, those routes will still point to the same components but will be carrying a different behavior "disabled animation" I was thinking if you can avoid the creation of additional stacks by setting option dynamically before you `navigate` or `reset` some thing like : 
```suggestion
     navigation.setOptions({
      animationEnabled: false
    })
```
I don't see any major issue with this approach just asking if we can optimize one step further.

Reply:
As long as there is a way to dynamically set options specifically without affecting the global behavior.

Reply:
Ok perfect. this was already a non-blocking feedback.